### PR TITLE
Move self-test for Genbank to unittest

### PR DESCRIPTION
**Pull request by souravsingh**
The PR fixes a part of #820 by moving the ``Bio.GenBank.Scanner`` self-test to a unitest.

### CODE REVIEW

File: Bio/GenBank/Scanner.py [hunk 1]
```diff
@@ -1653,374 +1653,5 @@ def _feed_misc_lines(self, consumer, lines):
...
-
-    print("")
-    print("EMBL Iteration")
-    print("==============")
-    e = EmblScanner()
-    for record in e.parse_records(StringIO(embl_example), do_features=True):
-        print("%s %s %s" % (record.id, record.name, record.description))
-        print(record.seq)
+    from Bio._utils import run_doctest
+    run_doctest(verbose=0)
```
Comment (peterjc) [inline Bio/GenBank/Scanner.py]: Are there any doctests in this file? If not, there is no reason to having these three lines of code.

Comment (souravsingh) [inline Bio/GenBank/Scanner.py]: There was a self test in the file, which I have migrated as it's own unittest.

Comment (peterjc) [inline Bio/GenBank/Scanner.py]: If there are no doctests, we don't need this:

```python
if __name__ == "__main__":
    from Bio._utils import run_doctest
    run_doctest(verbose=0)
```

**Comment by souravsingh** [comment]
@peterjc There are print statements in the unittest. Can you guide me through it?

**Comment by peterjc** [comment]
Basically replace the print lines with a unit test method checking something similar, e.g.

```python
        print("%s %s %s" % (record.id, record.name, record.description))
```

becomes:

```python
        self.assertEqual(record.id, "FILL IN THIS STRING")
        self.assertEqual(record.name, "FILL IN THIS STRING")
        self.assertEqual(record.description, "FILL IN THIS STRING")
```

while for:

```python
        print(record.seq)
```

you might just do something like this:

```python
        self.assertEqual(len(record.seq), 12345)
```

**Comment by peterjc** [comment]
You'll also need to make sure ``flake8`` is happy:

```
$ flake8 Bio/
Bio/GenBank/Scanner.py:1657:1: W293 blank line contains whitespace
Bio/GenBank/Scanner.py:1657:1: W391 blank line at end of file
```

**Pull request closed without merging**